sandy/errorr.py

```diff
@@ -315,7 +330,7 @@ def read_mf33(tape, mat, mt):
             GROW = L.N2
             GCOL = L.L2
             M[GROW-1, GCOL-1:GCOL+NGCOL-1] = L.B
-            if GCOL+NGCOL >= NG and GROW >= NG:
+            if GROW >= NG:
                 break
```

Comment:
I thought this was already fixed

Reply:
Yes it was in my previous PR. I think this and the conflicts in errorr comes from the fact I didn't branch directly from v1.0.
Maybe I'm creating the same branch but from v1.0 so we solve the issues. Can you reject this one. I'm opening a new PR following the standard procedure.